projects/lib/src/engineconfiguration.cpp

```diff
@@ -473,10 +473,10 @@ bool EngineConfiguration::operator!=(const EngineConfiguration& other) const
 
 static bool equivalent(const QStringList& l1, const QStringList& l2, Qt::CaseSensitivity cs = Qt::CaseSensitive)
 {
-	for (auto str : l1)
+    for (auto &str : l1)
```

Comment:
This changes semantics. Should use "const auto &" if you're not modifying the string. Anyway, this change doesn't look necessary.